src/events.rs

```diff
@@ -209,10 +210,11 @@ impl EventAssert {
 
 /// supervisor_started asserts an event that tells a supervisor with the given
 /// name started
-pub fn supervisor_started(input_name: &'static str) -> EventAssert {
+pub fn supervisor_started(input_name0: &str) -> EventAssert {
+    let input_name = input_name0.to_owned();
```

Comment:
These `to_owned` calls are actually unnecessary I think, since you are comparing against a ref value, and then you use this string in a `format!` which will do the copying you need.

Reply:
Removing the `to_owned` gives me a compilation error about lifetimes might not outlive the scope

```
error[E0621]: explicit lifetime required in the type of `input_name`
   --> src/events.rs:214:17
    |
213 |   pub fn supervisor_started(input_name: &str) -> EventAssert {
    |                                         ---- help: add explicit lifetime `'static` to the type of `input_name`: `&'static str`
214 |       EventAssert(Box::new(move |ev| match &ev {
    |  _________________^
215 | |         Event::SupervisorStarted(NodeData { runtime_name }) => {
216 | |             if runtime_name != &*input_name {
217 | |                 Some(format!(
...   |
225 | |         _ => Some(format!("Expecting SupervisorStarted; got {:?} instead", ev)),
226 | |     }))
    | |______^ lifetime `'static` required

error: aborting due to previous error

For more information about this error, try `rustc --explain E0621`.
error: could not compile `capataz`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error[E0621]: explicit lifetime required in the type of `input_name`
   --> src/events.rs:214:17
    |
213 |   pub fn supervisor_started(input_name: &str) -> EventAssert {
    |                                         ---- help: add explicit lifetime `'static` to the type of `input_name`: `&'static str`
214 |       EventAssert(Box::new(move |ev| match &ev {
    |  _________________^
215 | |         Event::SupervisorStarted(NodeData { runtime_name }) => {
216 | |             if runtime_name != &*input_name {
217 | |                 Some(format!(
...   |
225 | |         _ => Some(format!("Expecting SupervisorStarted; got {:?} instead", ev)),
226 | |     }))
    | |______^ lifetime `'static` required

error: aborting due to previous error

For more information about this error, try `rustc --explain E0621`.
```

That's why they were `'static` before